config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
+  resources :users
```

Comment:
leia sobre `resources`
[documentação](https://guides.rubyonrails.org/routing.html#crud-verbs-and-actions)

![Captura de Tela 2022-09-30 às 22 13 46](https://user-images.githubusercontent.com/8137319/193377439-3a2aec4d-b310-404a-91e4-f33b12a90b99.png)